README.md

```diff
@@ -97,6 +97,9 @@ Build against the shared libpng instead of using the builtin one. libpng 1.6+ hi
 - ```-DUSE_SYSTEM_FFMPEG=ON/OFF``` (default = *OFF*)  
 Build against the shared ffmpeg libraries instead of using the builtin patched version. Try this if the builtin version breaks the OpenGL renderer for you.
 
+- ```-DUSE_SYSTEM_ZLIB=ON/OFF``` (default = *OFF*)  
```

Comment:
Why default to off?

Reply:
Mainly because of Windows users. If it is set to ON by default, Windows users will have to turn it OFF or to use ZLIB_ROOT or to setup correctly zlib on their system (as defined in https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindZLIB.cmake#L62). Using OFF by default guarantees a working out-of-the-box experience.
For Linux users, because zlib is widely available as a system package, it is not a problem to have it ON by default.
An alternative way could be to use different default values based on the operating system but I am not really keen about this method.

Reply:
Its wierd ignoring system provided libs when building on linux, something you have been working to move away from recently with recent integration commits where system packaged/maintained versions are used. This feels like it goes against that. All the situations where system libs are not used is usually to work around some issues (ffmpeg, llvm) and I feel that should be the default. I understand the 'out-of-the-box' idea, but if you're building on windows with cmake, you really are not going to be concerned about setting some cmake variables. Is it possible to make the submodule build a fallback?

Reply:
I have a new idea. Instead of
```
if (NOT USE_SYSTEM_ZLIB AND NOT ZLIB_ROOT)
    add_subdirectory(3rdparty/zlib EXCLUDE_FROM_ALL)
    set(ZLIB_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/zlib" "${CMAKE_CURRENT_BINARY_DIR}/3rdparty/zlib" CACHE INTERNAL "")
    set(ZLIB_LIBRARY zlibstatic)
endif()
```
in CMakeLists.txt, what do you think about the following lines?
```
find_package(ZLIB QUIET)
if (NOT ZLIB_FOUND)
    add_subdirectory(3rdparty/zlib EXCLUDE_FROM_ALL)
    set(ZLIB_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/zlib" "${CMAKE_CURRENT_BINARY_DIR}/3rdparty/zlib" CACHE INTERNAL "")
    set(ZLIB_LIBRARY zlibstatic)
endif()
```
If the system package is not found, the submodule is used. This way, the option `USE_SYSTEM_ZLIB` will not be necessary, Windows users will (mostly) fallback to the submodule while Linux users will keep using the system package.

Reply:
That sounds like a good solution to me.

Reply:
I pushed the change.